ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -49,6 +49,29 @@ ScrollView {
                 }
             }
 
+            StyledText {
+                //% "Application Logs"
+                text: qsTrId("app-logs")
+                font.pixelSize: 15
+                color: Style.current.blue
+                topPadding: 23
+
+                MouseArea {
+                    anchors.fill: parent
+                    cursorShape: Qt.PointingHandCursor
+                    hoverEnabled: true
+                    onEntered: {
+                        parent.font.underline = true
```

Comment:
More QML-idiomatic approach is to prefer properties binding, since `MouseArea` has property `containsMouse`.
`parent.font.underline: mouseArea.containsMouse`
just give `MouseArea` an `id: mouseArea`.

Reply:
Remove all the `MouseArea` properties (which I copied from `AboutContainer.qml`) and replace with `id: mouseArea`?

Yesterday should practically be considered my first day with QML, so I will have a lot of dumb/basic questions in the near future. I read/wrote a little QML in May 2020, before that I had no experience with it, and since then I've barely looked at it. Thanks for your help while I get up to speed. 😄

Reply:
Since I wasn't completely sure what you meant, I've simply added `id: mouseArea`.

Reply:
We have little page here "QML Crash Course":
https://hackmd.io/@status-desktop/H12LYBzhO

If this is boring to read, one of us can conduct a quick technical webinar about QML. Just ask @iurimatias to schedule one.